src/main/java/somethingrandom/interfaceadapters/additem/AddItemView.java

```diff
@@ -0,0 +1,143 @@
+package somethingrandom.interfaceadapters.additem;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+
+public class AddItemView extends JPanel implements ActionListener, PropertyChangeListener {
+
+    private static JPanel panel = new JPanel();
+    private static JFrame frame = new JFrame();
+    private static JLabel itemTitle = new JLabel();
+    private static JTextField itemtitleInput = new JTextField(100);
+    private static JLabel itemDisc = new JLabel();
```

Comment:
Should this be `itemDesc` and `itemDescInput`? Maybe spell it out to `itemDescription`, `itemDescriptionInput`?